pymc/distributions/timeseries.py

```diff
@@ -174,11 +200,13 @@ def logp(
         """
 
         # Calculate initialization logp
+        init_logp = logprob.logp(init, value[..., 0])
```

Comment:
what happened to double counting init? Is that not an issue anymore?

Reply:
No, we added a check in `__new__` that `init` is not part of the model variables so it's logp is not counted anywhere else, only here. See my comment to your question above